diff --git a/node_modules/gatsby/index.d.ts b/node_modules/gatsby/index.d.ts
index a4c4e4b..869e957 100644
--- a/node_modules/gatsby/index.d.ts
+++ b/node_modules/gatsby/index.d.ts
@@ -1531,7 +1531,7 @@ type Send<T> = (body: T) => void
 /**
  * Gatsby API Function route response
  */
-export interface GatsbyAPIFunctionResponse extends ServerResponse {
+export interface GatsbyAPIFunctionResponse<T = any> extends ServerResponse {
   /**
    * Send `any` data in response
    */
